tests/integration/test_openapi.py

```diff
@@ -42,3 +42,10 @@ def test_openapi_endpoint():
     paths = payload["paths"]
     for endpoint in ("/readiness", "/liveness", "/v1/query", "/v1/feedback"):
         assert endpoint in paths, f"Endpoint {endpoint} is not described"
+
+
+def test_openapi_endpoint_head_method():
+    """Check if REST API allows HEAD HTTP method for endpoint with OpenAPI specification."""
+    response = client.head("/openapi.json")
```

Comment:
We are not implementing it, right? This comes from FastAPI itself. Does it make sense to test it though?

Reply:
I think yes, as these are integration tests. So it check  the integration with FastAPI and configuration there.

Reply:
I can't say I understand the benefit of such a test, but ok. :)